web_src/css/repo.css

```diff
@@ -2361,14 +2361,29 @@
 .stats-table {
   display: table;
   width: 100%;
+  margin: 6px 0;
+  border-spacing: 2px;
 }
 
 .stats-table .table-cell {
   display: table-cell;
+  border-radius: 4px;
 }
 
 .stats-table .table-cell.tiny {
-  height: 0.5em;
+  height: 8px;
+}
+
+.stats-table .table-cell:first-child {
+  border-radius: 4px 0 0 4px;
+}
+
+.stats-table .table-cell:last-child {
+  border-radius: 0 4px 4px 0;
+}
+
+.stats-table .table-cell:only-child {
+  border-radius: 4px;
```

Comment:
Don't we have variables we can use for that?

Reply:
We have but it makes no sense to alter them here because if you go too large, this happens:

<img width="183" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/d2dd3aaf-e6dc-4f0a-8047-326797fa26cf">

Reply:
That said, I can reduce 1 selector here.